features/storage/kvstore/conf/filesystem_no_rbp/mbed_lib.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "storage_filesystem_no_rbp",
+    "config": {
+        "filesystem": {
+            "help": "Options are default, FAT or LITTLE. If default value the filesystem is chosen by the blockdevice type",
+            "value": "default"
+        },
+        "blockdevice": {
+            "help": "Options are default, SPIF, DATAFASH, QSPIF or SD. If default the block device will be chosen by the defined component",
```

Comment:
"help": "Options are default, SPIF, DATAFLASH, QSPIF or SD. If default the block device will be chosen by the defined component",